test/assert.js

```diff
@@ -268,12 +268,46 @@ test('.same() should not mask RangeError from underlying assert', function (t) {
 	var b = new Circular();
 
 	t.throws(function () {
-		assert.notSame(a, b);
+		assert.notDeepEqual(a, b);
 	});
 
 	t.doesNotThrow(function () {
-		assert.same(a, b);
+		assert.deepEqual(a, b);
 	});
 
 	t.end();
 });
+
+test('.same() should log a deprecation notice', function (t) {
+	var calledWith;
+	var originalConsole = console.warn;
+	console.warn = function () {
```

Comment:
Turns out that when using `util.deprecate`, it actually logs to `console.log` which is fine, but I can't seem to get this test to work because when I reassign `console.log` it appears that my reassignment is never called. I'm not sure what's going on here... Any tips appreciated.

Alternatively, we can trust that `util.deprecate` works and not worry about writing tests for this case... ¯\\\_(ツ)_/¯